### ISSUE TITLE
"Genome resequencing" operation is a subclass of topic "Sequencing"

### ISSUE THREAD

**Issue by albangaignard** [edamontology/edamontology#775]
Operation classes should not inherit from Topic classes. 

```
:operation_3923 a owl:Class ;
    rdfs:label "Genome resequencing" ;
    :created_in "1.24" ;
    oboInOwl:hasDefinition """Laboratory experiment to identify the differences between a specific genome (of an individual) and a reference genome (developed typically from many thousands of individuals).

WGS re-sequencing is used as golden standard to detect variations compared to a given reference genome, including small variants (SNP and InDels) as well as larger genome re-organisations (CNVs, translocations, etc.).

ows re-sequencing of complete genomes of any given organism with high resolution and high accuracy.""" ;
    oboInOwl:hasExactSynonym "Resequencing" ;
    oboInOwl:hasHumanReadableId "Whole_genome_sequencing" ;
    oboInOwl:hasNarrowSynonym "Amplicon panels",
        "Amplicon sequencing",
        "Amplicon-based sequencing",
        "Highly targeted resequencing",
        "WGR",
        "WGRS",
        "Whole genome resequencing" ;
    rdfs:comment "Amplicon sequencing is the ultra-deep sequencing of PCR products (amplicons), usually for the purpose of efficient genetic variant identification and characterisation in specific genomic regions.",
        "Ultra-deep sequencing" ;
    rdfs:subClassOf :topic_3168 .
```

**Comment by wm75** [comment]
Has any conclusion been reached on this one?

**Comment by matuskalas** [comment]
This was fixed in 1028c944. Genome resequencing is since a topic (http://edamontology.org/topic_3923). This change wasn't yet part of the major release 1.25, but released in the following minor releases.
Does this help, @wm75 ?

Sorry for the messy issues, the #726 was closed without explanation, and this duplicate issue was left open.

**Comment by wm75** [comment]
perfect @matuskalas ! Makes a lot of sense now.
